### ISSUE TITLE
Support for Internet Explorer X

### ISSUE THREAD

**Issue by konklone** [18F/pulse#141]
* Which IE version are we targeting as a minimum?
* Do we need to fix anything for it?

Looking at https://analytics.usa.gov, I'm comfortable with any of IE9, IE10, or IE11 (**preferred**), depending on how much work is involved. We're targeting a relatively engaged, not-totally-mainstream audience here, even when it's members of the public. And if CIOs or their IT staff are using IE8 (which implies Windows XP) to get their work done, they have a bigger problem.

**Comment by konklone** [comment]
Adding to Sprint 3, as I think we need to at least do a :eyes: on this before calling v1.

**Comment by jtexnl** [comment]
My preferred testing tool was blocked on my GSA computer, so I checked the site using IE's developer tools. Having never used them before, I don't know how great they are, but if they're to be trusted, then the result is that IE 7 & 8 don't work at all. IE9 functions, though some elements (dropdown menus to select the number of items in a table) render strangely. Overall, though, functionality seems to be fine. Can anyone recommend a better way of testing this? Is there a way to emulate IE9 on an XP machine?

**Comment by konklone** [comment]
@jtexnl Based on the [state of the world](https://analytics.usa.gov/), I'm comfortable not supporting IE7 or IE8, but thank you for testing them. IE7 and IE8 only work on XP, and XP won't run IE9 and up. So if we get pushback from some agencies or CIO shops about not supporting IE7 or IE8, maybe respond by implying that the next public dashboard will measure the use of Windows XP inside federal agencies. :smiley: 

For testing IE9 up, these VMs may be helpful: http://dev.modern.ie/tools/vms/ If IE9 functions fine, that makes me optimistic about IE10 and IE11.

The IE9 dropdown rendering issue sounds super familiar. @juliaelman may have an idea of how to handle that.

**Comment by konklone** [comment]
And @jtexnl if the Windows 7 VMs are too heavy for what you've got at your disposal today, don't worry. In fact, if you can llready test IE9, and you can just install IE11 on your current computer somewhere, that's probably good. IE10 is [less common](https://analytics.usa.gov/) than either IE9 or IE11.

**Comment by jtexnl** [comment]
Believe it or not, IE9 is the default version for GSA machines!

**Comment by konklone** [comment]
> Believe it or not, IE9 is the default version for GSA machines!

Oh, I believe it.

**Comment by gbinal** [comment]
To confirm, I agree entirely with supporting IE9 and up.  

I just filled https://github.com/18F/pulse/issues/162 to suggest a popup for folks with IE 8 or earlier (as well as other out of date browsers).

**Comment by gbinal** [comment]
Also, I'm browsing IE9 using citrix and it's fine with me.  The small stuff like what @jtexnl found is acceptable to me.

**Comment by jtexnl** [comment]
I can confirm that IE 9, 10, and 11 all work. I also tested it on Safari, Opera, and Yandex. The issues I have found: 
1) The D3 renders slowly in IE 9 and 10, though that could be a VM issue. 
2) The arrow dropdown issue is present in all versions of IE. 
3) When viewing the site with Safari from a Windows environment, the dropdown menu for setting table size was not visible at all. When I tried the same thing from an OSX environment, the issue was gone. Again, might be an anomaly, but probably worth double-checking.
